_posts/2025-05-09-improve-opensearch-cluster-performance.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: post
+title: "Improve OpenSearch cluster performance by separating search and indexing workloads"
+authors:
+  - naarcha
+  - pudyodu
+date: 2025-05-09
+categories:
+  - technical-posts
+meta_keywords: OpenSearch, shard roles, indexing, search separation, performance tuning
```

Comment:
meta_keywords: OpenSearch cluster performance, index shards, search workloads, node roles, indexing and search separation, OpenSearch benchmark, performance tuning, optimizing your cluster